docs/excel/custom-functions-dialog.md

```diff
@@ -72,13 +75,13 @@ function getStock (ticker) {
         }, 1000);
       } else {
         _dialogOpen = true;
-        OfficeRuntime.displayWebDialog(url, {
+        Office.displayWebDialogOptions(url, {
```

Comment:
This doesn't match the [current ref doc page](https://docs.microsoft.com/en-us/javascript/api/office-runtime?view=office-js) Are the reference docs out of date?

Reply:
Yes, they will be updated later when the d.ts goes live (which it theoretically is but I need to check with Joe/Madhav)